--- a/src/Game/AI/Scripts/Plays/DefaultDefensive.cpp
+++ b/src/Game/AI/Scripts/Plays/DefaultDefensive.cpp
@@ -863,7 +863,7 @@
 
         if (fTrueConfidence2 <= fNotFacingSideline)
         {
-            asm { b _abo_keep2 }
+            goto _abo_keep2;
         }
         else
         {
@@ -900,7 +900,7 @@
 
         if (fTrueConfidence3 <= fNotSeparating)
         {
-            asm { b _abo_keep3 }
+            goto _abo_keep3;
         }
         else
         {
