diff --git a/src/Game/BasicStadium.cpp b/src/Game/BasicStadium.cpp
index 333fef44..8aabe6cd 100644
--- a/src/Game/BasicStadium.cpp
+++ b/src/Game/BasicStadium.cpp
@@ -619,7 +619,7 @@ void BasicStadium::LoadListOfGameplayInvisibleModels()
 {
     char szOptBin[80];
     nlSNPrintf(szOptBin, 0x50, "%s-GameplayInvisibleModels.bin", m_szBaseName);
-    unsigned long fileSize;
+    u32 fileSize;
     u32* pCur;
     u32* pData = (u32*)nlLoadEntireFile(szOptBin, &fileSize, 0x20, AllocateEnd);
     if (pData != NULL)
