diff --git a/src/Game/Debug/TimeRegions.cpp b/src/Game/Debug/TimeRegions.cpp
index 442f2463..c7f2258a 100644
--- a/src/Game/Debug/TimeRegions.cpp
+++ b/src/Game/Debug/TimeRegions.cpp
@@ -8,8 +8,6 @@
 #include "Game/Debug/FrameCounter.h"
 #include "NL/nlPrint.h"
 
-template <>
-cBaseCamera* nlDLRingGetStart<cBaseCamera>(cBaseCamera*);
 
 TimeRegion* pGamePlayTimeRegion;
 TimeRegion* pNISTimeRegion;
