--- a/src/Game/Audio/CrowdMood.cpp
+++ b/src/Game/Audio/CrowdMood.cpp
@@ -14,7 +14,9 @@
 extern GCAudioStreaming::AudioBufferMgr g_BufferMgr;
 static void ___blank(const char*, ...);
 static void UpdateTiming(float);
-extern "C" void sndStreamMixParameterEx(unsigned long stid, unsigned char vol, unsigned char pan, unsigned char span, unsigned char auxa, unsigned char auxb);
+#ifndef TARGET_PC
+extern "C" void sndStreamMixParameterEx(SND_STREAMID stid, unsigned char vol, unsigned char pan, unsigned char span, unsigned char auxa, unsigned char auxb);
+#endif
 
 class SoundStrToIDNode;
 class AudioLoader
@@ -400,12 +402,16 @@
             pan = (unsigned char)nlRandom(0x7F, &nlDefaultSeed);
             pBuf = g_CrowdAudio.pHeckleStream->m_Buffers[0];
             pBuf->m_Pan = pan;
+#ifndef TARGET_PC
             sndStreamMixParameterEx(pBuf->m_StreamId, pBuf->m_Volume, pBuf->m_Pan, pBuf->m_SurroundPan, 0, 0);
+#endif
 
             pan = (unsigned char)nlRandom(0x7F, &nlDefaultSeed);
             pBuf = g_CrowdAudio.pHeckleStream->m_Buffers[0];
             pBuf->m_SurroundPan = pan;
+#ifndef TARGET_PC
             sndStreamMixParameterEx(pBuf->m_StreamId, pBuf->m_Volume, pBuf->m_Pan, pBuf->m_SurroundPan, 0, 0);
+#endif
         }
     }
 }
